app-admin/chezmoi-bin/chezmoi-bin-2.58.0.ebuild

```
@@ -0,0 +1,37 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker
+
+DESCRIPTION="Manage your dotfiles across multiple diverse machines, securely."
+HOMEPAGE="https://www.chezmoi.io https://github.com/twpayne/chezmoi"
+
+SRC_URI="
+	amd64? ( https://github.com/twpayne/chezmoi/releases/download/v${PV}/chezmoi_${PV}_linux_amd64.deb )
+	x86? ( https://github.com/twpayne/chezmoi/releases/download/v${PV}/chezmoi_${PV}_linux_i386.deb )
+	arm64? ( https://github.com/twpayne/chezmoi/releases/download/v${PV}/chezmoi_${PV}_linux_arm64.deb )
+	arm? ( https://github.com/twpayne/chezmoi/releases/download/v${PV}/chezmoi_${PV}_linux_armel.deb )
+"
+
+S="${WORKDIR}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 arm arm64 x86"
```

```suggestion
KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```
NonsolvableDepsInDev 的报错可以用 unstable keywords 来解决。

实际上 overlay 中一般不会用 stable keywords，不过 gentoo-zh 这边没有禁止。